### DISCUSSION TITLE
Additional Complex number features

### DISCUSSION THREAD

**Discussion by jhendersonHDF** [HDFGroup/hdf5#4631] [General]
The following are additional features that have not yet been implemented:

- HDFView support
- Fortran support
- Tools (just h5dump likely) option for setting imaginary unit (i, j, etc.) 
  - Suggestion to use "--imaginary-unit" for option name
- Data transform capabilities
  - Suggestion to use letter 'z' for transformation expression variable
- Add support to and tests for the scaleoffset and nbit filters with complex numbers
- Build and test complex number examples
  - Examples were added, but are not currently built due to needing to distinguish between MSVC and other compilers

**Comment by qkoziol** [comment]
Anything needed in the C++ wrappers?  @bmribler

**Reply by bmribler** [reply]
I don't think so, @qkoziol.  It is in consideration for depletion.

**Comment by qkoziol** [comment]
Have you considered pinging the h5py team to ask for their feedback?

**Reply by jhendersonHDF** [reply]
Not really direct contact, but https://github.com/h5py/h5py/issues/2373.

**Comment by qkoziol** [comment]
It would be useful to add the arrry/compound -> complex conversions to h5repack.
